tags/README.md

```diff
@@ -40,9 +40,26 @@ CNCF TAGs are modelled on Kubernetes SIGs.  Differences are intended to be minim
+
+TAGs explicitly have no direct authority over CNCF projects. In particular, the creation of CNCF TAG’s does not change the existing, successfully practiced [charter](https://github.com/cncf/foundation/blob/main/charter.md) goal that "Projects.. will be ‘lightly’ subject to the Technical Oversight Committee".
+
+### Project Engagement
+
+Projects themselves may engage with the TAGs:
+
+* when exploring engagement within the cloud native ecosystem
+* prior to submitting their application for CNCF consideration
+* after submitting their application for CNCF consideration
+* for guidance in moving levels within the CNCF
+upon direction by the CNCF Technical Oversight Committee (TOC) or Linux Foundation staff
+
+Project engagement within the Cloud Native ecosystem is a vital way to raise project visibility as well as gain insights from technical leaders in the CNCF community for project maintainers.
+
+As the Technical Advisory Groups (TAGs) are an entry point for many projects to engage with the Cloud Native ecosystem, TAGs are encouraged to guide projects to join the TAGs, Working Groups, collaborate with other projects as well as explore various avenues to further engage with the greater community.
 
-The TAGs should strive to present the TOC with easily understandable and votable "propositions", each of which is supported by clear written evidence.  A proposition may be “to approve this project for incubation based on this [written due diligence investigation](https://github.com/cncf/toc/blob/main/process#due-diligence-creation-or-refresh)”, or “to approve this landscape document based on these clear goals and evidence that it achieves them”.  It is of utmost importance that the information and proposals provided to the TOC by TAGs be highly accurate and unbiased, driven by the goal to improve the CNCF as a whole, rather than benefit one project or company over another.  We believe that the rising tide lifts all boats, and that is our goal.
+### Technical Leadership
+The TAGs should strive to present the TOC with easily understandable and votable "propositions", each of which is supported by clear written evidence.  A proposition may be “to approve this project for incubation based on this “to approve this landscape document based on these clear goals and evidence that it achieves them”.  It is of utmost importance that the information and proposals provided to the TOC by TAGs be highly accurate and unbiased, driven by the goal to improve the CNCF as a whole, rather than benefit one project or company over another.  We believe that the rising tide lifts all boats, and that is our goal.
```

Comment:
```suggestion
The TAGs should strive to present the TOC with easily understandable and votable "propositions", each of which is supported by clear written evidence.  A proposition may be “to approve this project for incubation based on ..." or “to approve this landscape document under its clear goals and the evidence that it achieves them”.  It is of utmost importance that the information and proposals provided to the TOC by TAGs be highly accurate and unbiased, driven by the goal to improve the CNCF as a whole, rather than benefit one project or company over another.  We believe that the rising tide lifts all boats, and that is our goal.
```

Reply:
Proposition is a new term here, do we intend to differentiate it from recommendations?

Reply:
"Proposition" is similar to "proposal" and is the original language from v1.0 of the README. This can be reviewed/revised after upcoming discussions at KubeCon.